cmake/Modules/Findlibyuv.cmake

```diff
@@ -17,17 +17,30 @@
 #=============================================================================
 #
 
-find_package(PkgConfig QUIET)
```

Comment:
`Findlibyuv.cmake` was a bit weird: it calls pkg-config to find libyuv installed in system even when `AVIF_LOCAL_LIBYUV` is set. I made it more similar to other `Find*.cmake`.